setup.py

```diff
@@ -0,0 +1,22 @@
+import setuptools
+print('# open("requirements.txt").readlines()', open("requirements.txt").readlines())
+setuptools.setup(
+    name="executor-clip",
+    version="2.0",
+    author='Jina Dev Team',
+    author_email='dev-team@jina.ai',
+    description="Executors that encode images and text",
```

Comment:
this `description ` is wrong. This one only `encodes` images

Reply:
okay, I focus on image for now